app/controllers/api/api_controller.rb

```diff
@@ -101,6 +101,71 @@ def dashboard()
     end
   end
 
+  def mobile_dashboard()
```

Comment:
Mostly similar to the normal dashboard() function. Changes around messages and terror tracker. More changes to come, most likely.